AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -135,7 +133,7 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
     func addMeasurement(_ measurement: Measurement, toStreamWithID id: MeasurementStreamLocalID) throws {
         let stream = try context.existingObject(with: id.id) as! MeasurementStreamEntity
 
-        let newMeasurement = MeasurementEntity(context: context)
+        let newMeasurement = NSEntityDescription.insertNewObject(forEntityName: "MeasurementEntity", into: context) as! MeasurementEntity
```

Comment:
Why did that need to be changed?

Reply:
Paweł said to me that this is the correct known way of inserting and object to core data, that's the main reason for that 😄